service/src/main/java/bio/terra/axonserver/app/controller/AwsResourceController.java

```diff
@@ -141,7 +160,8 @@ public ResponseEntity<Void> putSageMakerNotebookStop(
   @Override
   public ResponseEntity<ApiNotebookStatus> getSageMakerNotebookStatus(
       UUID workspaceId, UUID resourceId) {
-    NotebookStatus notebookStatus = getNotebook(workspaceId, resourceId).getStatus();
+    NotebookStatus notebookStatus =
+        getNotebook(workspaceId, resourceId, AwsCredentialAccessScope.READ_ONLY).getStatus();
```

Comment:
I was thinking maybe we'd plumb an optional Access Scope through all of the `aws/notebook/` API commands and pass the Read Only status from the UI, rather than hard-coding specifically for the get status command.  That said, read only should always be sufficient for "get access" so this is probably OK.

Reply:
Hmm, I can't think of a case where we would want status to not be READ_ONLY and start/stop to not be WRITE_READ (or are they READ_ONLY as well?). I don't have a strong opinion though and could make this change to avoid the hardcoding if its cleaner.